svc/quiz_v2/room/default_room/room.go

```diff
@@ -138,7 +141,7 @@ LOOP:
 
 			if r.IsFull() {
 				log.Println("Room is full")
-
+				time.Sleep(r.quizLobbyLatency)
```

Comment:
putting `time.Sleep` directly inside `eventProcessor` is a bad idea
For this `r.quizLobbyLatency` amount of time, `eventProcessor` is blocked, means no new messages will be processed.
You can think of smth like this:
```
go func() {
   time.Sleep(r.quizLobbyLatency)
   r.st.SetStatus(status_transactor.RoomIsFullStatus)
}()
```